README.md

```diff
@@ -17,10 +17,10 @@ You can authenticate by creating an API token. Using such token, VS Code can sub
 
 1. <img src="assets/sideview.png" alt="cogwheel" width=30px;/> Dodona icon in the Activity Bar (icon in left column)
 2. `Open Settings`
-   ![API token](assets/api-token2.png)
+   ![Open settings](assets/open-settings.png)
 3. Paste the API token you just made in text box.
 4. In the left column you get an overview of the exercise series of the course (expand it if necessary).
```

Comment:
Can you change this to ` In the left column you get an overview of the exercises and series of the course (expand it if necessary).`?

Reply:
Like this?
```suggestion
4. In the left column you get an overview of the exercises and series of the course (expand it if necessary).
```

Reply:
Yes, thank you.